repos.d/managarm.yaml

```diff
@@ -0,0 +1,28 @@
+###########################################################################
+# Managarm
+###########################################################################
+
+- name: managarm_x86_64
+  type: repository
+  desc: Managarm x86_64
+  family: managarm
+  color: "97bde3"
+  minpackages: 240
+  default_maintainer: fallback-mnt-managarm@repology
+  sources:
+    - name: [ x86_64 ]
```

Comment:
If it's the only arch, you don't need [] and `{source}` substitutions later. `subrepo` may be removed as well.